src/datagrid/DataGrid.tsx

```diff
@@ -1123,6 +1141,22 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
         return <HideShowColumns columns={allColumns} updateColumns={this.updateColumns} />;
     }
 
+    // function to build selected row count
+    private buildSelectedRowCount(): React.ReactElement {
+        const {selectedRowCount, selectionType} = this.props;
+        const showSelectedRowsCount =
+            selectedRowCount && selectedRowCount > 0 && selectionType === GridSelectionType.MULTI ? true : false;
```

Comment:
Can we write it as follows?
selectedRowCount && selectedRowCount > 0 && selectionType === GridSelectionType.MULTI;

Reply:
@payald  I want showSelectedRowsCount to initialized to false if condition is failed. So keeping it as it is